go.sum

```diff
@@ -278,6 +278,8 @@ github.com/vmihailenco/msgpack v4.0.1+incompatible h1:RMF1enSPeKTlXrXdOcqjFUElyw
 github.com/vmihailenco/msgpack v4.0.1+incompatible/go.mod h1:fy3FlTQTDXWkZ7Bh6AcGMlsjHatGryHQYUTf1ShIgkk=
 github.com/vmware/vra-sdk-go v0.2.10 h1:nw1NSroINtw8lnaGUtTBuvDWEepBjEkTjEukYSqfJxM=
 github.com/vmware/vra-sdk-go v0.2.10/go.mod h1:W6OERth9ssa4FgvS98SOVusGgBNWJs+D4Hq/5ueTgIQ=
+github.com/vmware/vra-sdk-go v0.2.11 h1:QSnlKItvEmnhd9yfaCLesTlb/I928F/OhQ7LpaGljyI=
```

Comment:
Seeing multiple versions in this file. Do we need to run `go mod tidy`?

Reply:
Done.